lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -69,8 +69,12 @@ int MmeNasStateManager::initialize_state(const mme_config_t* mme_config_p) {
 
   // Allocate the local mme state
   create_state();
-
+#if !MME_UNIT_TEST
```

Comment:
Wouldn't be better for abstraction purposes to change instead of passing entire `mme_config_p` on `initialize_state()` to just pass 2 args, `persist_state_enabled` and `max_ues`, this will translate into a simpler dependency injection from unit testing point of view? This way we can also avoid using the #if directives to control this.

Reply:
well, I do not see how this would prevent using `#if` directives. Note that the idea is to have persist_state_enabled as true and `#if` directive here is needed for preventing a connection set up. Also, note that we are passing the pointer to the configs, which I think is better than passing each used config separately.